new/rfcXXX-dco.md

```diff
@@ -0,0 +1,71 @@
+
+    As a contributor to Chef's open source projects,
+    I want to attest that each contribution is under the terms of the Apache License,
+    so that it is clear that I am able to make the contribution.
+
+    As a maintainer of Chef's open source projects,
+    I want to see a developer certificate of origin in every contribution,
+    so that it is clear that the contribution is allowed.
+
+## Specification
+
+Effective October 1, 2016 ("the cutover date"), Chef will no longer require CLAs or CCLAs for contributions to its open source projects.  Rather, Chef is adopting the developer certificate of origin ("DCO") used by several other projects and overall smart people.  All commits prior to the cutover date shall fall under the CLA process and will require a signed contributor license agreement.
+
+The DCO is an attestation attached to every contribution made by every developer. In the commit message of the contribution, the developer simply adds a Signed-off-by statement and thereby agrees to the DCO, which you can find below or at [http://developercertificate.org/](http://developercertificate.org/).
+
+    Developer's Certificate of Origin 1.1
+
+    By making a contribution to this project, I certify that:
+
+    (a) The contribution was created in whole or in part by me and I
+        have the right to submit it under the open source license
+        indicated in the file; or
+
+    (b) The contribution is based upon previous work that, to the
+        best of my knowledge, is covered under an appropriate open
+        source license and I have the right under that license to   
+        submit that work with modifications, whether created in whole
+        or in part by me, under the same open source license (unless
+        I am permitted to submit under a different license), as
+        Indicated in the file; or
+
+    (c) The contribution was provided directly to me by some other
+        person who certified (a), (b) or (c) and I have not modified
+        it.
+
+    (d) I understand and agree that this project and the contribution
+        are public and that a record of the contribution (including
+        all personal information I submit with it, including my
+        sign-off) is maintained indefinitely and may be redistributed
+        consistent with this project or the open source license(s)
+        involved.
+
+## Downstream Impact
+
+* Currybot, part of the Supermarket, will no longer be necessary.
+* A small DCO bot will need to be added to each of Chef's open-source repositories.
+* A blog post and mailing list announcement will be made before the cutover.
+* This will have *no* impact on [the "obvious fix" rule](https://docs.chef.io/community_contributions.html#the-obvious-fix-rule).  Contributions that meet these criteria will not need a DCO but must include "obvious fix" in the commit message as outlined in [the "obvious fix" policy](https://docs.chef.io/community_contributions.html#the-obvious-fix-rule).
```

Comment:
We've never enforced that they must include it in the message, it was a request to make things easier to triage.